packages/base/src/widget.ts

```diff
@@ -1167,9 +1172,24 @@ export class DOMWidgetView extends WidgetView {
       this.el.removeAttribute('tabIndex');
     }
   }
+
+  /**
+   * @deprecated Use {@link luminoWidget} instead (Since 8.0).
+   */
+  get pWidget(): Widget {
+    if (!DOMWidgetView.deprecationWarningDisplayed) {
+      console.warn(
```

Comment:
This is going to put a *lot* of warnings in consoles of users that cannot do anything about it as soon as we release. Perhaps we can start off deleting this console.warn, with just docs saying it is deprecated, then up it to a console warning later to put more pressure on migrating?

Reply:
The intention of this code was to maximally show one warning per page load. I can't see how this would happen more often, maybe I'm missing something? If one message is still too much, I'm still fine with taking it out.

Reply:
Sorry, I meant that lots of users will have this warning, not that it would show up a lot of times on a single page. I think it's gentler to give developers a chance to migrate before showing warnings to their users.